python/cuxfilter/dashboard.py

```diff
@@ -27,7 +29,7 @@
 
 
 def _create_dashboard_url(notebook_url: str, port: int):
-    return f"http://{notebook_url}/proxy/{port}/"
+    return f"http://{notebook_url}/proxy/{port}/ "
```

Comment:
Is the trailing space needed for some reason?

Reply:
removed